asf_floorplan_interpreter/pipeline/evaluation/README.md

```diff
@@ -6,14 +6,25 @@ We also run a full pipeline evaluation using our evaluation dataset - a manually
 
 ## Model test metrics
 
-As of 23rd November 2023 the models perform with the following metrics.
+<!-- As of 23rd November 2023 the models perform with the following metrics.
 
 | Model                           | Precision (B) | Recall (B) | Precision (M) | Recall (m) |
 | ------------------------------- | ------------- | ---------- | ------------- | ---------- |
 | `window_door_config_yolov8m_wd` | 0.87          | 0.90       | 0.77          | 0.79       |
 | `room_config_yolov8m`           | 0.87          | 0.87       | 0.88          | 0.88       |
 | `staircase_config_yolov8m`      | 0.77          | 0.87       | 0.76          | 0.85       |
-| `room_type_config_yolov8m`      | 0.65          | 0.84       | 0.65          | 0.84       |
+| `room_type_config_yolov8m`      | 0.65          | 0.84       | 0.65          | 0.84       | -->
+
+As of 21st December 2023 the models perform with the following metrics.
```

Comment:
is it worth noting the threshold we used to get these results here, in case we come back to it later?